<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>516</width>
    <height>380</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <property name="leftMargin">
     <number>1</number>
    </property>
    <property name="topMargin">
     <number>1</number>
    </property>
    <property name="rightMargin">
     <number>1</number>
    </property>
    <property name="bottomMargin">
     <number>1</number>
    </property>
    <property name="spacing">
     <number>2</number>
    </property>
    <item row="0" column="0">
     <layout class="QGridLayout" name="gridLayout_6">
      <property name="spacing">
       <number>0</number>
      </property>
      <item row="0" column="7">
       <widget class="QPushButton" name="pushButton_10">
        <property name="text">
         <string>Печать</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>Дата С</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QDateEdit" name="de_date_from">
        <property name="calendarPopup">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>По</string>
        </property>
       </widget>
      </item>
      <item row="0" column="5">
       <widget class="QPushButton" name="pushButton_9">
        <property name="text">
         <string>Расчитать</string>
        </property>
       </widget>
      </item>
      <item row="0" column="3">
       <widget class="QDateEdit" name="de_date_to">
        <property name="calendarPopup">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="4">
       <spacer name="horizontalSpacer_3">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="0" column="6">
       <widget class="QPushButton" name="pb_calc_sum">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>В деньгах</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="1" column="0">
     <widget class="QTableWidget" name="tableWidget">
      <property name="font">
       <font>
        <family>Tahoma</family>
        <pointsize>8</pointsize>
       </font>
      </property>
      <property name="editTriggers">
       <set>QAbstractItemView::NoEditTriggers</set>
      </property>
      <property name="alternatingRowColors">
       <bool>true</bool>
      </property>
      <property name="selectionBehavior">
       <enum>QAbstractItemView::SelectRows</enum>
      </property>
      <attribute name="verticalHeaderVisible">
       <bool>false</bool>
      </attribute>
      <attribute name="verticalHeaderDefaultSectionSize">
       <number>20</number>
      </attribute>
      <attribute name="verticalHeaderMinimumSectionSize">
       <number>18</number>
      </attribute>
      <column>
       <property name="text">
        <string>Артикул</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Всего</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Брака</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>% брака</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Себест.</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Сумма</string>
       </property>
      </column>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton_9</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>ui_calc()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>327</x>
     <y>4</y>
    </hint>
    <hint type="destinationlabel">
     <x>324</x>
     <y>-95</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pb_calc_sum</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>ui_calc_sum()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>403</x>
     <y>10</y>
    </hint>
    <hint type="destinationlabel">
     <x>389</x>
     <y>-56</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_10</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>ui_print()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>471</x>
     <y>10</y>
    </hint>
    <hint type="destinationlabel">
     <x>468</x>
     <y>-125</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>ui_calc()</slot>
  <slot>ui_calc_sum()</slot>
  <slot>ui_print()</slot>
 </slots>
</ui>
